### PR TITLE
Slimit v0.8.1

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by agcopenhaver** [comment]
@conda-forge/staged-recipes is this good to go?

File: recipes/slimit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/rspivak/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "015fc1917c0eff834514fcedd4251275a3dacb10df1b49549277fe5e19de1ddf"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (beckermr) [inline recipes/slimit/meta.yaml]: Can this package be `noarch: python`? See the documentation for more information: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Comment (agcopenhaver) [reply recipes/slimit/meta.yaml]: This package uses 2to3, so I don't believe so

File: recipes/slimit/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  imports:
+    - slimit
+    - slimit.tests
+
+about:
+  home: "https://github.com/rspivak/slimit"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "LICENSE"
+  summary: "SlimIt is a JavaScript minifier written in Python. It compiles JavaScript into more compact code so that it downloads and runs faster."
```
Comment (jakirkham) [inline recipes/slimit/meta.yaml]: You may consider wrapping this to improve readability.

Comment (agcopenhaver) [reply recipes/slimit/meta.yaml]: I'll fix it in the merged repo. Thanks for catching that

**Pull request merged**